guides/extensions/push-notifications/index.md

```diff
@@ -0,0 +1,3 @@
+# Push Notifications plugin for Aries Framework JavaScript
+
+The `push-notifications` package provides a simple way to implement push notications in a client application. It collects events triggered by the agent via AFJ into simple APIs that you can use to trigger push notiifcations in your client application.
```

Comment:
Should we add a link to the setup page, or do you think the sidebar will guide the user?

Reply:
Also can we include the full package name instead of just push-notifications? Or write it as a name so the Push Notifications package

Reply:
You got it. On it.